app/ parsers/bulkrax/xml_etd_dc_parser.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Bulkrax
+  class XmlEtdDcParser < XmlParser
+    def entry_class
+      Bulkrax::XmlEtdDcEntry
+    end
+
+    # @todo not yet supported
```

Comment:
This is from bulkrax proper. we have a ticket about files so I added it just incase it was relevant. I'll change it from a todo to a note since that gives the wrong impression.